src/eos-google-chrome

```diff
@@ -56,8 +56,8 @@ class GoogleChromeLauncher:
             logging.info("Flatpak launcher for Chrome found. Launching...")
             self._run_chrome_app(chrome_launcher, self._params)
         else:
-            logging.info("Could not find flatpak launcher for Chrome. Opening App Center...")
-            self._run_app_center_for_chrome()
+            logging.info("Could not find flatpak launcher for Chrome. Running installation script...")
+            self._install_chrome()
```

Comment:
This change does not seem right. The logic here so far is that something always happens when you click on the Chrome icon: either Chrome is run, if installed, or GNOME Software is opened in the GS page to allow the user to **manually** install it.

But with this change, the logic has changed: the eos-google-chrome-installer script will check the gsetting controlling the initial setup and either:
  * If the new gsetting key is enabled, it will open GNOME Software and start installing Google Chrome automatically
  * If the new gsetting key is disabled, it will do nothing.

It seems to me that the main problem here is that the gsetting key is encoding the wrong behaviour: the proposal (mentioned by @rshuler recently, IIRC) was to have a GSetting **to control the automatic installation behaviour**, but what the current code is doing is controlling whether GNOME Software is opened or not, and when it's opened it will always install automatically Google Chrome.

I think one way to fix this would be to simply keep the two scripts completely independent (assuming some duplication), or maybe add some extra flag to the installer script to distinguish between a manual and an automatic mode (or maybe a "normal" and a "initial-setup" mode if it makes more sense), and only check the gsetting value in the automatic case.

How does it sound?